features/networking/multus.feature

```diff
@@ -1533,3 +1533,59 @@ Feature: Multus-CNI related scenarios
       | bash | -c | ip -4 -brief a |
     Then the output should contain:
       | 192.168.42.4 |
+
+  # @author weliang@redhat.com
+  # @case_id OCP-41789
+  @admin
+  Scenario: [BZ1944678] Whereabouts IPAM CNI duplicate IP addresses assigned to pods	
+    Given the multus is enabled on the cluster
+    And I store all worker nodes to the :nodes clipboard
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    Given I obtain test data file "networking/multus-cni/NetworkAttachmentDefinitions/Bug-1944678.yaml"
+    When I run oc create as admin over "Bug-1944678.yaml" replacing paths:
+      | ["metadata"]["namespace"] | <%= project.name %> |
+    Then the step should succeed
+    
+    # Create a pod absorbing above net-attach-def
+    Given I obtain test data file "networking/multus-cni/Pods/generic_multus_pod.yaml"
+    When I run oc create over "generic_multus_pod.yaml" replacing paths:
+      | ["metadata"]["name"]                                       | macvlan-bridge-pod1 |
+      | ["metadata"]["annotations"]["k8s.v1.cni.cncf.io/networks"] | macvlan-bridge      |
+      | ["spec"]["containers"][0]["name"]                          | macvlan-bridge      |
+    Then the step should succeed
+    And the pod named "macvlan-bridge-pod1" becomes ready
+    # Check the created pod has correct ip
+    When I execute on the "macvlan-bridge-pod1" pod:
+      | bash | -c | ip -4 -brief a |
+    Then the output should contain:
+      | 10.199.199.100 |
```

Comment:
Thanks @weliang1. Are we sure that multus would assign dynamic IPs in sequential order as we are intentionally checking 100,101 and 102? Normally DHCP takes a random IP from the specified range, AFAIK.

Reply:
Oh I see , (from above description)2. Make sure the pods' IP should be assigned in the order from that range.

Reply:
> Thanks @weliang1. Are we sure that multus would assign dynamic IPs in sequential order as we are intentionally checking 100,101 and 102? Normally DHCP takes a random IP from the specified range, AFAIK.

@anuragthehatter So far, the multus/whereabouts assign the ip addresses to the pods in sequential order, the testing results approved this in several multus test cases.